Quadriga/src/main/webapp/WEB-INF/views/settings/addpublicpage.jsp

```diff
@@ -103,43 +103,42 @@
 
 				<tr>
 					<td style="width: 170px">Title *</td>
-					<td style="width: 400px"><input path="title" size="60"
-						class="titleValue" id="title1" value="${publicpageObject0.title}" /></td>
-					<td><div id="title_err"></div></td>
-					<td><form:errors path="title" class="ui-state-error-text"></form:errors></td>
+					
+					<td style="width: 600px"><input path="title" size="60" 
```

Comment:
Didn't we talk about this? the column shouldn't have any width. Bootstrap should take the complete window width on its own.